movai_core_shared/logger.py

```diff
@@ -30,67 +30,6 @@
     datefmt=LOG_FORMATTER_DATETIME,
 )
 
-LOG_FORMATTER_HTTP = logging.Formatter(
-    "[%(levelname)s][%(asctime)s][%(module)s][%(funcName)s][%(lineno)d]: %(message)s",
-    datefmt=LOG_FORMATTER_DATETIME,
-)
-
-
-class HealthNodeHandler(logging.handlers.HTTPHandler):
```

Comment:
you deleted this class instead of changing it

Reply:
well this has been done by Ofer, from what I see this class was actually got replaced by RemoteLogger which is found inside movai_core_enterprise.

[](https://github.com/MOV-AI/movai-core-enterprise/blob/feature-BP-859-create-AlertsHandler-class-for-manager-dispacher/movai_core_enterprise/message_client_handlers/remote_logger.py)

Reply:
got it